index.html

```diff
@@ -221,9 +225,85 @@ <h1>Skills</h1>
                 <li>gitlab</li>
                 <li>Terminal</li>
               </ul>
-            </div>
           </section>
-          </div>
-          </body>
-          </html>
-         
\ No newline at end of file
+          <section class="touch">
+            <h2>
+              I'm always interested in hearing about new projects, so if you'd like
+              to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] I kindly suggest that you create your account on formspree.io so that you can get your unique `{form_id}` as the above will not work without a form_id.
- [ ] I kindly suggest that you confirm that the form is submitted successfully with valid input values.